README.md

```diff
@@ -105,7 +111,7 @@ a version number bump.
 
 ## Who maintains this Module?
 
-This Module is maintained by [Gruntwork](http://www.gruntwork.io/). If you're looking for help or commercial
+This Module is maintained by [Gruntwork](http://www.gruntwork.io/?ref=repo_aws_vault). If you're looking for help or commercial
```

Comment:
Oh, just noticed the "who maintains this module" section. Since this is no longer true, we should probably remove this from each of the READMEs.